### DISCUSSION TITLE
请问如何配置pdf和png的下载目录？Windows系统。

### DISCUSSION THREAD

**Discussion by hongsedaima** [lencx/ChatGPT#450] [General]
如题，确实不太懂，虽然知道配置文件，但不知道怎么配置。在chat.download.json中写了个新目录，但是发现生成pdf或png的时候还是保存到了默认目录，而这个默认目录确认不太好找就是了。

另外，当内容太长时，下载的pdf有问题，文字被纵向压缩了——都很胖。

**Comment by lencx** [comment]
它目前并不支持配置，而且下载的文件在 Control Center 中统一进行管理，你可以在那里找到它们。

关于 PDF 变形的问题，暂时没有时间处理，你可以尝试保存图片。
